docroot/themes/custom/yukonca_glider/templates/field/field--node--field-primary-item-blocks.html.twig

```diff
@@ -38,31 +38,31 @@
  */
 #}
 {%
-  set classes = [
+    set classes = [
     'primary-items',
     'row d-block',
     'field',
     'field--name-' ~ field_name|clean_class,
     'field--type-' ~ field_type|clean_class,
     'field--label-' ~ label_display,
     label_display == 'inline' ? 'clearfix',
-  ]
+]
 %}
 {%
-  set title_classes = [
+    set title_classes = [
     'field__label',
     label_display == 'visually_hidden' ? 'visually-hidden',
-  ]
+]
 %}
 
 <div{{attributes.addClass(classes,'field__items')}}>
```

Comment:
Same here. Move class to the variable and add spaces.
Please fix these issues in all your twig files :)